tomopy/misc/corr.py

```diff
@@ -61,6 +61,7 @@
 import logging
 import numexpr as ne
 import concurrent.futures as cf
+import tomocuda
```

Comment:
probably also do a check on the GPU available and fall back to cpu if a usable one is not found?